lib/gridfs-stream/index.js

```diff
@@ -295,20 +283,7 @@ GridFSBucket.prototype.openDownloadStreamByName = function(filename, options) {
  */
 
 GridFSBucket.prototype.rename = function(id, filename, callback) {
-  if (typeof callback === 'function') {
-    return _rename(this, id, filename, callback);
-  }
-
-  var _this = this;
```

Comment:
just curious, what would the point of doing this have been?

Reply:
using `_this`? It's the same idea as `var self = this;` you're just using a locally scoped version of `this` for use within a closure.